pygfx/__init__.py

```diff
@@ -26,7 +26,7 @@
 
 del pylinalg
 
-__version__ = "0.3.0"
+__version__ = "0.4.0"
 version_info = tuple(map(int, __version__.split(".")))
 
 __wgpu_version_range__ = "0.15.1", "0.17.0"
```

Comment:
will wgpu need to be pinned because of #816

Reply:
Yes, good point!  I'm on holiday now. I may or may not find the time to fix this :)